docs/basics/test-config-annotation/test-annotation.md

```diff
@@ -16,7 +20,7 @@ export const Highlight = ({children, color}) => ( <span style={{
     }}>{children}</span> );
 
 
-Test annotation refers to adding information to your tests after they have completed, such as setting a name, build number, tag, and Pass/Fail status. These annotations are useful for managing your tests and builds (for example, when searching and sorting tests in your [Archives](/test-results/archived-test-results)). You can add annotations with our [REST API](/basics/test-config-annotation/test-annotation) or the [Selenium JavaScript Executor](/basics/test-config-annotation/test-annotation). You can also use [sample test frameworks](https://github.com/saucelabs-training) to automatically add annotations to your tests.
+Test annotation refers to adding information to your tests after they have completed, such as setting a name, build number, tag, and Pass/Fail status. These annotations are useful for managing your tests and builds (for example, when searching and sorting tests in your [Archives](/test-results/archived-test-results)). You can add annotations with our [REST API](/basics/test-config-annotation/test-annotation/#sauce-labs-rest-api) or the [Selenium JavaScript Executor](/basics/test-config-annotation/test-annotation/#selenium-javascript-executor). You can also use [sample test frameworks](https://github.com/saucelabs-training) to automatically add annotations to your tests.
```

Comment:
> Test annotation refers to adding information to your tests after they have completed,....

This is only correct for the API, the JS executor can only be used **during** a test. Can we clarify this better?

Reply:
Secondly we might want to mention that the update-job API can only be used for VDC, not for RDC

Reply:
I added a "Virtual Devices Only" badge to clarify.